### DISCUSSION TITLE
Welcome to 24hr-Beginner-Projects Discussions!

### DISCUSSION THREAD

**Discussion by rakshaa2000** [Superposition-Chennai/24hr-Beginner-Projects#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AK16092003** [comment]
Hello, I am Ashwanth 1st yr of NITT, I know only the basics of python, c, SQL, Java, HTML, CSS, js, PHP, android studio. 
I don't know any advanced topics.
Can I know which project can I choose to start with ..., This is going to be my first open-source contribution.
